backend/database.js

```diff
@@ -152,6 +155,9 @@ export default async () => {
           }
         }
         await writeData(key, value)
+        Object.keys(prefixHandlers).forEach(prefix => {
+          cache.delete(prefix + key)
+        })
```

Comment:
I do not understand what this is or why it's here...

1. Are you certain this is correct?
2. If so, could you please explain why it's correct in a comment above?

Reply:
See https://github.com/okTurtles/group-income/commit/b62d291d4b0fe211bcc49a422401b39c968611c0